views/shopping-cart.pug

```diff
@@ -0,0 +1,29 @@
+extends layout
+block content
+    div.card
+        div.card-header
+            h2 Your shopping cart items
+        div.card-body
+            ul
+                if foodItems
+                    each item in foodItems
+                        li
+                            h6=item.item.name
+                            span.btn.btn-success.disabled="$"+item.price+".00"
```

Comment:
ℹ️ Will item prices always be in whole USD's? Consider there are many libraries designed to address the concern of formatting Money strings for user's comprehension that also makes sense to computers

> $1,000 vs 1000